lib/src/metta/runner/stdlib.rs

```diff
@@ -194,7 +194,11 @@ impl Grounded for IncludeOp {
         let program_text = String::from_utf8(program_buf)
             .map_err(|e| e.to_string())?;
         let parser = crate::metta::text::OwnedSExprParser::new(program_text);
-        context.push_parser(Box::new(parser));
+        //QUESTION: do we want to do anything with the result from the `include` operation?
```

Comment:
I would say it should behave like included file is a part of the modules itself. Thus user which includes module B from module A I think will expect the results are returned as they are results of the A. To make it behave absolutely identical we need to implement it differently. Included it on a level of parser I for example. Thus may be we can leave the code as is while we don't have a real usecase.